scripts/git-secrets-check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+if [ -d /tmp/git-secrets ]; then
+    echo The command git-secrets is available
+else
+    echo The command git-secrets is not available, cloning...
+    git clone git@github.com:awslabs/git-secrets.git /tmp/git-secrets/
+fi
+
+# ensure we have correct configs in place
+[ -f ../.gitconfig ]
+cat ../.gitconfig >> .git/config 
```

Comment:
In general I'm not a huge fan of relative paths in scripts because they tend not to be as portable. In this case it looks to be pretty reliable as these paths are relative to the script itself so as long as it isn't moved it should be fine.

Is this intended to be run by developers as well or just during CI?

Reply:
This runs just during CI.